clang/lib/Basic/Targets/X86.h

```diff
@@ -983,6 +983,7 @@ class LLVM_LIBRARY_VISIBILITY CygwinX86_64TargetInfo : public X86_64TargetInfo {
       : X86_64TargetInfo(Triple, Opts) {
     this->WCharType = TargetInfo::UnsignedShort;
     this->WIntType = TargetInfo::UnsignedInt;
+    this->UseMicrosoftManglingForC = true;
```

Comment:
Are you sure this and `CCK_MicrosoftWin64` are correct?  I thought these were not involved on MinGW either?

Reply:
Oops, it looks like `UseMicrosoftManglingForC` is set in `WindowsTargetInfo` (which applies to MinGW too), while `CCK_MicrosoftWin64` is set in `MicrosoftX86_64TargetInfo` (which is not)